test/functionalTest/cases/3884_ip_len_in_logs/ip_len_in_logs.test

```diff
@@ -0,0 +1,155 @@
+# Copyright 2021 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Service path is not ignored in GET registrations operations
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0-1 IPV4
+brokerStart CP1
+
+--SHELL--
+#
+# 01. Register entity E1 for CP1 in CB
+# 02. Create E1 in CP1
+# 03. GET /entities/E1 from CB to provoke a forwarded request to CP1, to provoke the bug
+# 04. GET /entities/E1 from CP1 to make sure it-s still alive
```

Comment:
Maybe a 05 step could be added to get the CB logs and check that from= is "sane" (i.e. without the problems described at https://github.com/telefonicaid/fiware-orion/issues/3884#issuecomment-862642537)

Reply:
Can easily be done, but ... having a functest depending on the output in the logfile is not the best of ideas.
I'll have to pick a specific logline (grep it from the logfile), where we know the FROM_IP has been set, and then compare.
It that logline is removed or changed, then the functest will stop working.
We've had plenty of problems with that in the past.

Are you sure you want this?

Reply:
Using grep and awk we can be very precise and get only the piece of log we are interested in, i.e. the line with the from field due to the "X-Forwarded-For" we are using, looking only for the from= field in that line. We have some examples of that in the .test (looking for "awk" should show some cases).

However, it is not a must for this PR. I can work on it after merging this PR (I have some recent experience with that).

Reply:
I'm aware. Kind of invented it.
What I mean is, if the line disappears or is modified, the whole thing starts failing.
In my opinion it's a bad idea.
But, I can see you want it, so I'll add it

Reply:
Fixed in b4857fd